### ISSUE TITLE
Strobe Media Playback: invalid parameter passed to method

### ISSUE THREAD

**Issue by Marc477** [denivip/osmf-hls-plugin#112]
I tried both Strobe Media Playback and Grindplayer with your HLS plugin, it works fine on localhost, but if I put the .swf on another domain, I get this error (invalid parameter passed to method) with  Strobe Media Playback , and nothing is working with Grindplayer.

This is the only thing i change, I know that the files get loaded (network tab) but then it crashes. I know also that it is not a crossdomain.xml issue since i've put that file on my server (it works with video.js).

 I use chrome 42 and windows 7,  Flash 17.0.0.169

I think you can get a similar issue here:http://osmfhls.kutu.ru/  (SMP player only)

![error](https://cloud.githubusercontent.com/assets/7688794/7425064/3fd766fc-ef78-11e4-902b-372a6200b66d.png)

Thank you

**Comment by dangerden** [comment]
@Marc477 could you please give a link to your test page, i'll check what goes wrong there

**Comment by Marc477** [comment]
Test page: http://marcantoinedesbiens.ca/test/

Both video have the same parameters , but the second loads the plugin from another domain. Only the first works.

**Comment by Marc477** [comment]
And yes it is exactly the same swf file on both servers

**Comment by D3adKnight** [comment]
Test page don't load both players.

GET http://zenterac.com/data/StrobeMediaPlayback.swf net::ERR_CONNECTION_TIMED_OUT
GET http://zenterac.com/data/StrobeMediaPlayback.swf net::ERR_CONNECTION_TIMED_OUT

**Comment by Marc477** [comment]
Yes, it is because my test server was down, i changed the test so that now the files are hosted on google drive. You can retry here:

http://marcantoinedesbiens.ca/test/

Thank you for your help

**Comment by D3adKnight** [comment]
I was trying to reproduce this error but all works fine with(or without) crossdomain.
You can see this test page: http://denivip.ru/crossdomain_hls/
Player on the denivip.ru domain, plugin loaded from your zenterac.com domain.

**Comment by Marc477** [comment]
Thank you so much, with your example I could compare the response headers.
On my first test server (node.js): these response headers were missing:
Content-Length:163648
Content-Type:application/x-shockwave-flash

On my second test, for some reason loading HLSDynamicPlugin.swf  from https (google drive) wasnt working. Only StrobeMediaPlayback.swf was loading. (Maybe StrobeMediaPlayback.swf does not support loading plugins from HTTPS)

So it was 2 different errors on the 2 server i tried. But it wasnt a crossdomain issue.

**Comment by Marc477** [comment]
But now when I load StrobeMediaPlayback.swf from another domain, the javascript callback doesnt seem to work:

javascriptCallbackFunction: "FHLS_Event"

http://marcantoinedesbiens.ca/test/

I tried from server 1 -> server 2, and from server 2 -> server 1.
So I dont think it is a header issue this time.

**Comment by D3adKnight** [comment]
Try this:
```
{
   allowFullScreen: "true",
   wmode: wmodeValue,
   allowScriptAccess: "always" // add this line in embed params
}

**Comment by Marc477** [comment]
Thank you for your help !
